src/docs/asciidoc/resources/migration_guide_to_3.11.adoc

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2022 RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// This document is subject to the terms of the Creative Commons Attribution 4.0 International license.
+// If a copy of the license was not distributed with this
+// file, You can obtain one at https://creativecommons.org/licenses/by/4.0/.
+// SPDX-License-Identifier: CC-BY-4.0
+
+= Migration Guide from release 3.10.x to release 3.11.0
+
+== Card publication 
+OperatorFabric will now check user perimeter when publishing cards. To disable perimeter validation it is possible to set the configuration parameter `checkPerimeterForCardSending` to `false` in cards-publication service configuration file. 
```

Comment:
via endpoint /cards (it does not concern user cards which was always controlled)